Mention wasm-features in `Readme.md` (#167)
argmin-rs/argmin: README.md

ndarray-linalg = { version = "*", features = ["intel-mkl-static"] }
```

> #### Required for WebAssembly

Reviewer: ```suggestion
#### Experimental support for compiling to WebAssembly
```

> To prevent panics when compiling to WASM, one of the following features needs to be used:

Reviewer: ```suggestion
When compiling to WASM, one of the following features must be used:
```

```toml
argmin = { version = "0.4.7", features = ["wasm-bindgen"] }
```

```toml
argmin = { version = "0.4.7", features = ["stdweb"] }
```

> 

Reviewer: ```suggestion
Note that WASM support is still experimental. Please report any issues you encounter when compiling argmin to WASM.

```
